src/components/header/Header.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import headerBackground from './header-background.svg';
+import Logo from '../../assets/logo.svg';
+import LogoName from '../../assets/logoname.svg';
+
+class Header extends Component {
+
+  render() {
+    return (
+      <div className="header-div">
+        <img id="header-background" alt="colorful overlapping shapes background" src={headerBackground}/>
```

Comment:
Nit: Let's capitalize the first letter of the alt text

Reply:
Fixed!